logout/index.html

```diff
@@ -0,0 +1,334 @@
+        <div class="header-container">
+          <span class="h4 text-wt-bold">Filters</span>
+          <span class="text-size-sm"
+            ><a href="" class="link text-dec-none">Clear All</a></span
+          >
+        </div>
+        <div class="categories">
+          <p>Categories</p>
+          <ul class="category-list">
+            <li>
+              <label for="Alldevices" class="text-size-sm"
+                ><input type="checkbox" name="All devices" id="" />All devices
+                <span>(99)</span></label
+              >
+            </li>
+            <li>
+              <label for="Phones" class="text-size-sm"
+                ><input type="checkbox" name="Phones" id="" />Phones
+                <span>(99)</span></label
+              >
+            </li>
+            <li>
+              <label for="Tablets" class="text-size-sm"
+                ><input type="checkbox" name="Tablets" id="" />Tablets
+                <span>(99)</span></label
+              >
+            </li>
+            <li>
+              <label for="Laptops" class="text-size-sm"
+                ><input type="checkbox" name="Laptops" id="" />Laptops
+                <span>(99)</span></label
+              >
+            </li>
+            <li>
+              <label for="Pre-built PCs" class="text-size-sm"
+                ><input type="checkbox" name="Pre-built PCs" id="" />Pre-built
+                PCs <span>(99)</span></label
+              >
+            </li>
+            <li>
+              <label for="Accessories" class="text-size-sm"
+                ><input type="checkbox" name="Accessories" id="" />Accessories
+                <span>(99)</span></label
+              >
+            </li>
+          </ul>
+        </div>
+        <div class="ratings">
+          <p>Ratings</p>
+          <ul class="list-style-none">
+            <li>
+              <label for="" class="text-size-sm">
+                <input type="radio" name="rating" id="" />⭐⭐⭐⭐+
+              </label>
+            </li>
+            <li>
+              <label for="" class="text-size-sm">
+                <input type="radio" name="rating" id="" />⭐⭐⭐+
+              </label>
+            </li>
+            <li>
+              <label for="" class="text-size-sm">
+                <input type="radio" name="rating" id="" />⭐⭐+
+              </label>
+            </li>
+            <li>
+              <label for="" class="text-size-sm">
+                <input type="radio" name="rating" id="" />⭐+
+              </label>
+            </li>
+          </ul>
+        </div>
+        <div class="filter-by-price">
+          <p>Filter by price</p>
+          <div class="price-slider">
+            <span class="text-size-sm">1000</span>
+            <span class="text-size-sm">5000</span>
+            <span class="text-size-sm">10000</span>
+            <span class="text-size-sm">20000+</span>
+            <input type="range" name="price-slider" id="" />
```

Comment:
generally, the range slider has the minimum and highest price only. Also, the range slider is not wide enough for the extreme ends.

Reply:
Yes I'm in the process of fixing this.